### PR TITLE
Add contributing guide

### PR DESCRIPTION
**Pull request by ThetaSinner**
Adding a guide with what I believe to be the simplest steps required to get started building on the Scaffolding and contributing back a PR

### CODE REVIEW

**Review by guillemcordoba — APPROVED** [review]

File: guides/contributing.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+```
+
+and see a version number like `holochain_scaffolding_cli 0.1.11`.
+
+To run the tests, run the provided script:
+
+```bash
+./run_tests.sh
+```
+
```
Comment (steveej) [inline guides/contributing.md:45]: do we want to give the alternative of compiling `hc-scaffold` as it's done on CI?

````suggestion

#### Using `nix develop`
Alternatively you can replicate how CI compiles and introduces `hc-scaffold` to its PATH and run the test script via an ad-hoc nix environment with the following command:

```
nix develop --override-input "versions/scaffolding" . .#ci --command ./run_test.sh`
```
````

File: guides/contributing.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+cd scaffolding
+nix develop
+```
+
+You should now see your terminal prompt change to look something like `[rustDev:~/source/holo/scaffolding]$`.
+
+You are now set up, you can start making changes or check the next section to learn how to run the tests.
+
+### Running the scaffolding tests
+
```
Comment (steveej) [inline guides/contributing.md:25]:
```suggestion

#### Using `cargo install`
```

**Review by steveej — APPROVED** [review]
thanks :raised_hands: i also suggest showing how CI runs the tests.

**Comment by ThetaSinner** [comment]
Yes good suggestion @steveej, changes accepted!

**Pull request merged**